### ISSUE TITLE
Problem with backup export products in cvs

### ISSUE THREAD

**Issue by bitsjohmorales** [woocommerce/woocommerce#28928]
Please provide us with the information requested in this bug report. Without these details, we won't be able to fully evaluate this issue. 
Bug reports lacking detail, or for any other reason than to report a bug, may be closed without action.

**Prerequisites (mark completed items with an [x]):**
- [ ] I have have carried out troubleshooting steps and I believe I have found a bug.
- [ ] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

**Describe the bug**

**Expected behavior**

**Actual behavior**
A clear and concise description of what actually happens. Please be as descriptive as possible; 

**Steps to reproduce the bug (We need to be able to reproduce the bug in order to fix it.)**
Steps to reproduce the bug:

**Screenshots**

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
We use the [WooCommerce System Status Report](https://docs.woocommerce.com/document/understanding-the-woocommerce-system-status-report/) to help us evaluate the issue. 
Without this report we won't be able to fully evaluate this issue.
<details>
```
The System Status Report is found in your WordPress admin under **WooCommerce > Status**. 
Please select “Get system report”, then “Copy for support”, and then paste it here.
```
</details>

**Comment by zhongruige** [comment]
Hi @bitsjohmorales ,

Thank you for submitting the issue. However, you didn’t fill out the details of the bug report template that we ask for. Without these details, we can’t fully evaluate this issue. Please provide us with the information requested so we could take a look further.

**Describe the bug**
A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Expected behavior**
A clear and concise description of what you expected to happen.

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by ramjamx** [comment]
I can confirm theres is a problem while export products on woocomerce 4.9.2. wordpress 5.6 

Prerequisites (mark completed items with an [x]):

[x]) I have have carried out troubleshooting steps and I believe I have found a bug.
[x])I have searched for similar bugs in both open and closed issues and cannot find a duplicate.
Describe the bug
The export process is not complete, instead redirection to posts list.


Expected behavior
It has to create a CVS with products in woocomerce.

Actual behavior
There is no CVS created, instead redirection to post lists.

To reproduce the bug 

Go to 'Products'
Click on 'Export products'
Fill The necessary fields 
There is an error on developer console 
https://domain.xyz/wp-admin/edit.php?post_type=product&page=product_exporter ERROR 500

And two warnings
The resource https://www.domain.xyz/wp-content/plugins/woocommerce/packages/woocommerce-admin/dist/app/index.js?ver=1.8.3 was preloaded using link preload but not used within a https://www.domain.xyz/wp-admin/edit.php?post_type=product&page=product_exporter 
few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

The resource https://www.domain.xyz/wp-content/plugins/woocommerce/packages/woocommerce-admin/dist/components/index.js?ver=1.8.3 was preloaded using link preload but not used within a https://www.domain.xyz/wp-admin/edit.php?post_type=product&page=product_exporter 
few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally.

After that no cvs creation ocurred.

Screenshots
https://pasteboard.co/JLZMeLl.png

Isolating the problem (mark completed items with an [x]):

[x]) I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
[x]) This bug happens with a default WordPress theme active, or Storefront.
 [x]) I can reproduce this bug consistently using the steps above.
WordPress Environment
Wordpress system status:
https://pasteboard.co/JLZN9R6.png

**Comment by zhongruige** [comment]
Hi @ramjamx,

Thank you for raising the issue and providing those details. However, I can’t reproduce it on my end.

1. I set up a WooCommerce store and imported the default products:
<img width="1489" alt="Screen Shot 2021-02-17 at 3 52 36 PM" src="https://user-images.githubusercontent.com/71906536/108278380-3074cb80-7138-11eb-8882-3412d7601e5b.png">
2. I clicked on "Export" and configured it with the following:
<img width="727" alt="Screen Shot 2021-02-17 at 3 50 03 PM" src="https://user-images.githubusercontent.com/71906536/108278416-3ec2e780-7138-11eb-886b-25b323e38e54.png">
3. The CSV was exported with all products, as expected:
<img width="2033" alt="Screen Shot 2021-02-17 at 3 50 30 PM" src="https://user-images.githubusercontent.com/71906536/108278723-a7aa5f80-7138-11eb-86b7-435775903064.png">

I tested the above on the following:
* WordPress 5.6.1
* WooCommerce 5.0

It could be worth updating the WooCommerce version to see if the error still persists.
